database/migrations/2024_08_01_194616_publishers_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('publishers', function (Blueprint $table) {
+            $table->id();
+            $table->string('name')->unique();
+            $table->string('address');
+            $table->string('website');
+            $table->string('email')->unique();
+            $table->string('phone', 20);
```

Comment:
@radosevic-dr Default 20?

Reply:
I chose the number 20 since the usual max character limit for telephone numbers is 17, but providers may allow longer numbers. Should I remove it or put other length?

Reply:
@radosevic-dr I actually think it's 15, but nevermind, don't worry about this